bin/mixserv_daemon.sh

```diff
@@ -134,14 +168,14 @@ case $1 in
     if [ -f $MIXSERV_PID_FILE ]; then
```

Comment:
I think that the current implementation does the same thing with `yarn application -status` when SIGTERM/SIGINT received.
https://github.com/myui/hivemall/pull/246/files#diff-b1e95d2afdb57b4e66a66403f081634fR339